### PR TITLE
Add KVIrc

### PR DESCRIPTION
**Pull request by farribeiro**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** http://www.kvirc.net/?id=mailinglist
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* The patch is exclusive to latest release, not necessary in the future release

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by farribeiro** [comment]
bot, build net.kvirc.KVIrc

File: net.kvirc.KVIrc.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+rename-icon: kvirc
+rename-desktop-file: kvirc.desktop
+command: kvirc
+
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=xdg-download
+  - --persist=$XDG_CONFIG_HOME/KVIrc
```
Comment (barthalion) [inline net.kvirc.KVIrc.yaml:14]: As mentioned in the previous PR, this is not going to work. It should be `--persist=.config/KVIrc`.

Comment (farribeiro) [reply net.kvirc.KVIrc.yaml:14]: Okay. Thanks.

Changes in the way

**Comment by barthalion** [comment]
bot, build net.kvirc.KVIrc

File: net.kvirc.KVIrc.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+app-id: net.kvirc.KVIrc
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-icon: kvirc
+rename-desktop-file: kvirc.desktop
+command: kvirc
+
+finish-args:
```
Comment (barthalion) [inline net.kvirc.KVIrc.yaml:9]:
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri, suffix _dri)
libGL error: failed to load driver: amdgpu
```

You should add `--device=dri`.

Comment (farribeiro) [reply net.kvirc.KVIrc.yaml:9]: Curious in the nvidia driver not fails

Comment (farribeiro) [reply net.kvirc.KVIrc.yaml:9]: Thinking may a GL extension installed?

Comment (farribeiro) [reply net.kvirc.KVIrc.yaml:9]: Removed the GL and opens with this error:

```
➜  ~ flatpak run net.kvirc.KVIrc                                                              
Gtk-Message: 15:16:34.105: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:16:34.105: Failed to load module "canberra-gtk-module"
Qt: Session management error: None of the authentication protocols specified are supported
qt.glx: qglx_findConfig: Failed to finding matching FBConfig for QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize -1, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
No XVisualInfo for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize -1, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
Falling back to using screens root_visual.
qt.glx: qglx_findConfig: Failed to finding matching FBConfig for QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize 0, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
No XVisualInfo for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize 0, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
Falling back to using screens root_visual.
qt.glx: qglx_findConfig: Failed to finding matching FBConfig for QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize -1, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
No XVisualInfo for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize -1, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
Falling back to using screens root_visual.
```

Comment (hfiguiere) [reply net.kvirc.KVIrc.yaml:9]: not sure what you are getting at here.

**Comment by barthalion** [comment]
bot, build net.kvirc.KVIrc

**Pull request closed without merging**